TokenAction.php

```diff
@@ -16,18 +16,22 @@
  */
 class TokenAction extends \yii\base\Action
 {
+    /** Format of response
+     * @var string
+     */
+    public $format = Response::FORMAT_JSON;
 
     public $grantTypes = [
         'authorization_code' => 'conquer\oauth2\granttypes\Authorization',
         'refresh_token' => 'conquer\oauth2\granttypes\RefreshToken',
 //         'client_credentials' => 'conquer\oauth2\granttypes\ClientCredentials',
-//         'password' => 'conquer\oauth2\granttypes\UserCredentials',
+         'password' => 'conquer\oauth2\granttypes\UserCredentials',

```

Comment:
Вендор править не нужно, это включается в настройках акшина, в контроллере, где этот акшн подключается. т.е. там нужно будет определить этот массив grantTypes с новым способом.
В рекомендациях https://tools.ietf.org/html/rfc6749#section-4.3 есть вот такое: 
Since this access token request utilizes the resource owner's
   password, the authorization server MUST protect the endpoint against
   brute force attacks (e.g., using rate-limitation or generating
   alerts).
По-хорошему, нужно добавить описание как правильно подключать этот способ. RateLimiter уже есть во фреймворке, нужно рассказать как его включить, какие тут могут быть проблемы  и т.п. Т.е. взять и просто добавить поддержку интерфейса - это слишком просто. Лично мне ссыкотно, что такой способ будет открыт по умолчанию...